linera-storage/src/db_storage.rs

```diff
@@ -330,11 +340,12 @@ pub struct DbStorage<Database, Clock = WallClock> {
 #[derive(Debug, Serialize, Deserialize)]
 enum RootKey {
     ChainState(ChainId),
-    CryptoHash(CryptoHash),
+    ConfirmedBlock(CryptoHash),
     Blob(BlobId),
     Event(ChainId),
-    BlockExporterState(u32),
+    Placeholder,
     NetworkDescription,
+    BlockExporterState(u32),
 }
```

Comment:
I would actually base the RootKey on whatever data is available for partitioning:
```
enum RootKey {
    NetworkDescription,
    BlockExporterIndex(u32),
    ChainId(ChainId),
    CryptoHash(CryptoHash),
    BlobId(BlobId),
}
```